pkg/cmd/cli/cmd/startbuild.go

```diff
@@ -337,3 +375,52 @@ func gitRefInfo(repo git.Repository, dir, ref string) (buildapi.GitRefInfo, erro
 	info.Message = lines[4]
 	return info, nil
 }
+
+// WaitForBuildComplete waits for a build identified by the name to complete
+func WaitForBuildComplete(c osclient.BuildInterface, name string) error {
```

Comment:
Yes, there is another occurrence in extended tests... we should move this to some common helper, will do as follow up.